src/pages/TradeV3/perps/DepositWithdraw.tsx

```diff
@@ -478,10 +308,87 @@ const CloseTradingAccount: FC<{ setDepositWithdrawModal: Dispatch<SetStateAction
             </div>
           )}
         </SELECTED_COIN>
-      </Dropdown>
+      </Dropdown> */}
+      <DropdownMenu>
+        <DropdownMenuTrigger asChild={true}>
+          <Button variant="outline" colorScheme="default" className={cn('w-full mt-2 rounded-[100px]')}>
```

Comment:
shouldn't need to do w-full, pass the prop `fullWidth` would an additional prop for 100% roundness help?